intake/source/cache.py

```diff
@@ -44,12 +54,17 @@ def _ensure_cache_dir(self):
 
     def _munge_path(self, cache_subdir, urlpath):
         import re
+
+        regex = os.path.normpath(strip_http(self._spec['regex']))
+        path = os.path.normpath(strip_http(urlpath))
```

Comment:
Is `normpath` necessary? What failed without it? Worry that only local FS it makes no difference, but remote FS might require original key without munge (or is this only for local files in the cache?)

Reply:
I'm using it here to remove trailing slashes. :-)

Reply:
Also, notice I'm calling `normpath` on the striped `regex` which http is the only case that will require the protocol in the regex to match properly.  However, you're right about the `urlpath`.  I'll fix that.